File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Grid;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {

        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            GridRecentItemsHelper helper = new GridRecentItemsHelper(gridView1);
            helper.RecentItems.Add(new GridRecentItem("WindowsFormsApplication1", WindowsApplication1.Properties.Resources.vs2010, false));
            helper.RecentItems.Add(new GridRecentItem("WindowsFormsApplication2", WindowsApplication1.Properties.Resources.vs2010, true));
            helper.RecentItems.Add(new GridRecentItem("WindowsFormsApplication3", WindowsApplication1.Properties.Resources.vs2010, false));
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Views.Grid

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim helper As New GridRecentItemsHelper(gridView1)
			helper.RecentItems.Add(New GridRecentItem("WindowsFormsApplication1", My.Resources.vs2010, False))
			helper.RecentItems.Add(New GridRecentItem("WindowsFormsApplication2", My.Resources.vs2010, True))
			helper.RecentItems.Add(New GridRecentItem("WindowsFormsApplication3", My.Resources.vs2010, False))
		End Sub
	End Class
End Namespace